addons/hr_expense/tests/test_expenses.py

```diff
@@ -211,17 +211,14 @@ def test_expense_main_flow(self):
             (self.analytic_account_1 | self.analytic_account_2).unlink()
 
         # Unlinking moves
-        (payment_1 | payment_2).move_id.button_draft()
+        (payment_1 | payment_2).action_draft()
         self.assertRecordValues(expense_sheet_by_employee, [{'payment_state': 'not_paid', 'state': 'post'}])
         expense_sheet_by_employee.account_move_ids.button_draft()
         expense_sheet_by_employee.account_move_ids.unlink()
 
         with self.assertRaises(UserError, msg="For company-paid expenses report, deleting payments is an all-or-nothing situation"):
-            expense_sheet_by_company.account_move_ids[:-1].button_draft()
-            expense_sheet_by_company.account_move_ids[:-1].unlink()
+            expense_sheet_by_company.account_move_ids[:-1].origin_payment_id.unlink()
         expense_sheet_by_company.account_move_ids.origin_payment_id.unlink()
-        expense_sheet_by_company.account_move_ids.button_draft()
-        expense_sheet_by_company.account_move_ids.unlink()
```

Comment:
Same

Reply:
The asserts are actually indirect in the following assertRecordValues :) but I don't have any objection against adding a tighter grip on them

Reply:
I have added assertFalse after the unlink, they will be put on the next push :smile: